app/src/main/java/com/celzero/bravedns/ui/DNSConfigureWebViewActivity.kt

```diff
@@ -209,18 +209,18 @@ class DNSConfigureWebViewActivity : AppCompatActivity(R.layout.activity_faq_webv
     private fun updateLocalStamp() {
         Log.i(LOG_TAG, "Webview: Local stamp has been set from webview - $receivedStamp")
         if (receivedStamp.isEmpty() || receivedStamp == Constants.BRAVE_BASE_STAMP) {
-            val localStamp = persistentState.getLocalBlockListStamp()
+            val localStamp = persistentState.localBlockListStamp
             if (localStamp.isNotEmpty()) {
                 return
             }
-            persistentState.setLocalBlockListStamp("")
+            persistentState.localBlockListStamp = ""
```

Comment:
I don't get this code... 

if received-stamp is empty or equal to base stamp then:
1. set local-stamp to the one in persistent-state (?)
2. if local-stamp, which now equals whatever is in persistent-state, is not empty, then return
3. Else, set persistent-state to empty string (? but isn't it empty if it gets to this line in the first place?)

Reply:
Removed the code. I should gave returned if its empty or the equal to the base.